client/my-sites/importer/importer-medium.jsx

```diff
@@ -0,0 +1,45 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import PureRenderMixin from 'react-pure-render/mixin';
+
+/**
+ * Internal dependencies
+ */
+import FileImporter from './file-importer';
+
+/**
+ * Module variables
+ */
+const importerData = {
+	title: 'Medium',
+	icon: 'medium'
+};
+
+export default React.createClass( {
+	displayName: 'ImporterMedium',
+
+	mixins: [ PureRenderMixin ],
+
+	render: function() {
+		importerData.description = this.translate(
+			'Import posts, tags, images and videos ' +
+			'from a Medium export file.'
+		);
+
+		importerData.uploadDescription = this.translate(
+			'Upload a {{b}}Medium export file{{/b}} to start ' +
+			'importing into {{b2}}%(title)s{{/b2}}.'
+			, {
```

Comment:
It was probably the network trouble I was having earlier that prevented my comment from posting here - we don't use the leading-comma, so we should probably move this up to the previous line